slick.grid.js

```diff
@@ -710,44 +726,44 @@ if (typeof Slick === "undefined") {
       return _getContainerElement(getCanvases(), columnIdOrIdx, rowIndex);
     }
 
-    function getActiveCanvasNode(element) {
-      setActiveCanvasNode(element);
+    function getActiveCanvasNode(e) {
 
-      return $activeCanvasNode[0];
-    }
+      if(e === undefined)
+        return _activeCanvasNode;
```

Comment:
wrap in curly brace please :) ... or return on a single line, that could also be acceptable. There are a few of these in this file

Reply:
Done